extensions/sql-database-projects/package.json

```diff
@@ -133,6 +133,12 @@
         "command": "sqlDatabaseProjects.openContainingFolder",
         "title": "%sqlDatabaseProjects.openContainingFolder%",
         "category": "%sqlDatabaseProjects.displayName%"
+      },
+      {
+        "command": "sqlDatabaseProjects.expandAllNodes",
+        "title": "%sqlDatabaseProjects.expandAllNodes%",
+        "category": "%sqlDatabaseProjects.displayName%",
+        "icon": "$(expand-all)"
```

Comment:
Not necessarily in this PR but we will have a colapse all too right? 
once we do we will need to change icons to toggle based on state OR we can have both at all times but one enabled one disabled...

Reply:
There is actually a collapse all included in this PR; it is included as an option when the tree view is created (in extensions/sql-database-projects/src/controllers/mainController.ts).  This results in two buttons in the viewlet at all times. 

I know it's a little strange to have collapse all created via an option vs expand all created manually. Did this because expand all cannot be created as an option, and couldn't find APIs to create collapse all manually. Any recommendations on this?